GlyphExtractorTest added

szwedj opened the pull request:
I have assumed that:
- White 1x1 BufferedImage should be identified to have 0 lines and 0 letters
- One line BufferedImage created with ImageGenerator should have 2 lines (one as newline) - is that correct?
- Test file with 20 actual lines should be identified to have 22 lines (one as title separator and one as newline at the end) - is that correct?

When counting letters results are incorrect (higher than actual number). Please see results of testCutLettersWithLines.

Please let me know in case tests need any enhancements.

tomcug commented:
> White 1x1 BufferedImage should be identified to have 0 lines and 0 letters
Correct

> One line BufferedImage created with ImageGenerator should have 2 lines (one as newline) - is that correct?
Just 1 line

> Test file with 20 actual lines should be identified to have 22 lines (one as title separator and one as newline at the end) - is that correct?
Only 20 lines

szwedj commented:
I have changed assert values in accordance to @tomcug guidelines.
I have also added tests with single-pixel glyphs before those generated with ImageGenerator.

@tomcug please review the change.
